github_issue_summarization/ks-kubeflow/components/params.libsonnet

```diff
@@ -53,17 +53,18 @@
     ui: {
       namespace: "null",
       githubToken: "",
+      image: "gcr.io/kubeflow-examples/issue-summarization-ui:v20180629-v0.1-2-g98ed4b4-dirty-182929",
     },
     "tfjob-v1alpha2": {
-      name: "tfjob-v1alpha2",
-      image: "gcr.io/kubeflow-dev/tf-job-issue-summarization:v20180425-e79f888",
+      name: "tfjob-issue-summarization",
+      image: "gcr.io/kubeflow-examples/tf-job-issue-summarization:v20180629-v0.1-2-g98ed4b4-dirty-182929",
       input_data_gcs_bucket: "kubeflow-examples",
       input_data_gcs_path: "github-issue-summarization-data/github-issues.zip",
       output_model_gcs_bucket: "kubeflow-examples",
-      output_model_gcs_path: "github-issue-summarization-data/output_model.h5",
+      output_model_gcs_path: "github-issue-summarization-data",
       sample_size: "100000",
-      gcpSecretName: "user-gcp-sa",
-      gcpSecretFile: "user-gcp-sa.json",
+      gcpSecretName: "gcp-credentials",
```

Comment:
These aren't the right defaults. "user-gcp-sa" and "user-gcp-sa.json" are the defaults as set by our GKE deployment instructions.